src/components/AddressField/AddressField.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {
```

Comment:
Would you mind importing this component directly from the distributable directory of `decentraland-ui`?